contracts/test/accounts/GlobalAdmin.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.6.11;
+
+import { IMapleGlobals } from "../../interfaces/IMapleGlobals.sol";
+
+contract GlobalAdmin {
+
+    /************************/
+    /*** Direct Functions ***/
+    /************************/
```

Comment:
I don't mind this, but I worry the separation of "Direct Functions" and "Try Functions" can result the function and try version of the function being too distant. No big deal though.